packages/react/src/components/ComboButton/ComboButton.stories.js

```diff
@@ -53,17 +53,154 @@ export const WithIcons = () => (
+        <MenuItem label="Third action" />
+        <MenuItem label="Fourth action" disabled />
+      </ComboButton>
+
+      <ComboButton label="Bottom right" menuAlignment="bottom-right">
+        <MenuItem label="Second action with a long label description" />
+        <MenuItem label="Third action" />
+        <MenuItem label="Fourth action" disabled />
+      </ComboButton>
+    </div>
+
+    <div
+      style={{
+        display: 'flex',
+        marginTop: '15rem',
+        justifyContent: 'space-between',
+      }}>
+      <ComboButton label="Top" menuAlignment="top" tooltipAlignment="bottom">
+        <MenuItem label="Second action with a long label description" />
+        <MenuItem label="Third action" />
+        <MenuItem label="Fourth action" disabled />
+      </ComboButton>
+
+      <ComboButton
+        label="Top left"
+        menuAlignment="top-left"
+        tooltipAlignment="bottom">
+        <MenuItem label="Second action with a long label description" />
+        <MenuItem label="Third action" />
+        <MenuItem label="Fourth action" disabled />
+      </ComboButton>
+
+      <ComboButton
+        label="Top right"
+        menuAlignment="top-right"
+        tooltipAlignment="bottom">
+        <MenuItem label="Second action with a long label description" />
+        <MenuItem label="Third action" />
+        <MenuItem label="Fourth action" disabled />
+      </ComboButton>
+    </div>
+  </>
+);
+
 export const Playground = (args) => {
+  console.log('combobutton', args);
   const onClick = action('onClick (MenuItem)');
 
+  const changeButtonPostion = () => {
+    const selectedValue = document.getElementById('select').value;
+    const mainDiv = document.getElementById('mainDiv');
+    mainDiv.removeAttribute('style');
+    switch (selectedValue) {
+      case 'top center':
+        mainDiv.style.justifyContent = 'center';
+        mainDiv.style.display = 'flex';
+        break;
+      case 'top left':
+        mainDiv.style.justifyContent = 'flex-start';
+        mainDiv.style.display = 'flex';
+        break;
+      case 'top right':
+        mainDiv.style.justifyContent = 'flex-end';
+        mainDiv.style.display = 'flex';
+        break;
+      case 'bottom left':
+        mainDiv.style.justifyContent = 'flex-start';
+        mainDiv.style.position = 'absolute';
+        mainDiv.style.insetBlockEnd = 0;
+        mainDiv.style.insetInlineStart = '2rem';
+        mainDiv.style.marginBlock = '2rem';
+        mainDiv.style.marginInline = '2rem';
+        break;
+      case 'bottom right':
+        mainDiv.style.justifyContent = 'flex-end';
+        mainDiv.style.position = 'absolute';
+        mainDiv.style.insetBlockEnd = 0;
+        mainDiv.style.insetInlineEnd = '2rem';
+        mainDiv.style.marginBlock = '2rem';
+        mainDiv.style.marginInline = '2rem';
+        break;
+      case 'center':
+        mainDiv.style.position = 'fixed';
+        mainDiv.style.top = '50%';
+        mainDiv.style.left = '50%';
```

Comment:
Should this use [`inset`](https://developer.mozilla.org/en-US/docs/Web/CSS/inset)? I think the syntax would be something like:

```suggestion
        mainDiv.style.inset = '50% auto auto 50%';
```

Reply:
We are gonna delete this code from Playground. It was just for testing purpose